mintpy/timeseries2velocity.py

```diff
@@ -601,6 +637,28 @@ def layout_hdf5(out_file, atr, model):
         ds_name_dict[dsName+'Std'] = [dataType, (length, width), None]
         ds_unit_dict[dsName+'Std'] = 'm'
 
+    # time func 4 - exp
+    for i in range(num_exp):
+        # dataset name
+        dsName = 'exp{}'.format(model['exp'][0][i])
```

Comment:
Thinking about the overlay of multiple taus for the fitting for different layerings or timescales, as Mark pointed out, I would imagine it has multiple exponential functions with the same onset time and different taus, thus, I think we should add the tau info to the `dsName` for unique identification. Maybe `exp{YYYYMMDD}Tau{char_time}`?

Same comments for the log dsName as well.

Reply:
Ah, that what Mark meant when he talked about it. I see now. Thanks for explaining. I will make the changes allowing for the same onset time with different char_time(s) for both exp and log funcs.

Reply:
Updated in the new commit. Thanks.